File: rest/call/list-get-example-4/list-get-example-4.5.x.php
<?php
// Get the PHP helper library from https://twilio.com/docs/libraries/php
require_once '/path/to/vendor/autoload.php'; // Loads the library
use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/user/account
// To set up environmental variables, see http://twil.io/secure
$sid = getenv('TWILIO_ACCOUNT_SID');
$token = getenv('TWILIO_AUTH_TOKEN');
$client = new Client($sid, $token);

$calls = $client->calls->read(
    array(
        "status" => "in-progress",
        "startTimeAfter" => "2009-07-04",
        "startTimeBefore" => "2009-07-06"
    )
);
// Loop over the list of calls and echo a property for each one
foreach ($calls as $call) {
    echo $call->to;
}

File: rest/recording/list-get-example-4/list-get-example-4.5.x.rb
require 'twilio-ruby'

# put your own credentials here
# To set up environmental variables, see http://twil.io/secure
account_sid = ENV['TWILIO_ACCOUNT_SID']
auth_token = ENV['TWILIO_AUTH_TOKEN']

# set up a client to talk to the Twilio REST API
@client = Twilio::REST::Client.new(account_sid, auth_token)

# Loop over recordings and print out a property for each one
@client.recordings.list(
  date_created_before: Time.new('2016-10-15'),
  date_created_after: Time.new('2016-10-12')
).each do |recording|
  puts recording.duration
end
